lib/engines/content_block_manager/app/views/content_block_manager/content_block/editions/host_content/preview.html.erb

```diff
@@ -1,7 +1,7 @@
 <% content_for :page_title, "Preview content block in host document" %>
 <% content_for :context, "Preview #{@content_block_edition.block_type.humanize.downcase}" %>
 <% content_for :title, @preview_content.title %>
-<% content_for :title_margin_bottom, 3 %>
+<% content_for :title_margin_bottom, 1 %>
```

Comment:
this is currently being ignored, but should be correct once this PR is approved https://github.com/alphagov/whitehall/pull/9938